### PR TITLE
Isoseq modules

### PR DESCRIPTION
**Pull request by sguizard**
The lima program accept bam file as input, but also fasta, fasta.gz, fastq, fastq.gz, consensusreadset.xml.

### CODE REVIEW

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Nice! 
Just a small question is the `alz.ccs.consensusreadset.xml` not missing?

**Comment by sguizard** [comment]
Thanks for the reviewing. 
I decided to removed it from the dataset. 
This metadata xml file include full path to data files. 
So, the test wont be able to run on this format.

File: README.md [hunk 1]
```diff
@@ -150,9 +150,15 @@ If you cannot find suitable test data on this repository, please contact us on t
...
     * fasta: 
+      * alz.ccs.fasta: CCS reads generated using pbccs on alz.bam in fasta format
+      * alz.ccs.fasta.gz: CCS reads generated using pbccs on alz.bam in gziped fasta format
       * primers.fasta: NEB Clonetech primers 
     * fastq:
+      * alz.ccs.fastq: CCS reads generated using pbccs on alz.bam in fastq format
+      * alz.ccs.fastq.gz: CCS reads generated using pbccs on alz.bam in gziped fastq format
       * test_hifi.fastq.gz: Reads mapping to a randomly selected contig from the whole genome assembly by [Cheng et al., 2021](https://www.nature.com/articles/s41592-020-01056-5) of the child of the GIAB Ashkenazim trio [RM8392](https://www-s.nist.gov/srmors/view_detail.cfm?srm=8392). The reads were taken from [SRR10382244](https://www.ncbi.nlm.nih.gov/sra/?term=SRR10382244).
+    * xml:
+      * alz.ccs.consensusreadset.xml: CCS reads generated using pbccs on alz.bam in consensusreadset.xml
```
Comment (JoseEspinosa) [inline README.md:161]: Then it should be removed from the readme, right?
```suggestion


```

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sguizard) [reply README.md]: Indeed, I forgot to remove it from the Readme. 
Sorry for that.

**Review by JoseEspinosa — APPROVED** [review]
🚀

**Pull request merged**